src/microbe_stage/editor/MicrobeEditorGUI.cs

```diff
@@ -1000,6 +1005,17 @@ public void UpdateMicrobePartSelections()
         }
     }
 
+    /// <summary>
+    ///   Updates the visibility of the current action cancel button.
+    /// </summary>
+    public void UpdateCancelButtonVisibility()
+    {
+        cancelButton.Hide();
+
+        if (editor.MovingOrganelle != null)
+            cancelButton.Show();
```

Comment:
This still needs improvement. I think an else for hiding it here would be better than unnecessarily calling hide here as it doesn't make this code any simpler